src/test/resources/sql/scheduledPrisonerReturnFromCourt_init.sql

```diff
@@ -0,0 +1,139 @@
+INSERT INTO OFFENDERS (OFFENDER_ID, OFFENDER_NAME_SEQ, ID_SOURCE_CODE, LAST_NAME, NAME_TYPE, FIRST_NAME,
+                                 MIDDLE_NAME, BIRTH_DATE, SEX_CODE, SUFFIX, LAST_NAME_SOUNDEX, BIRTH_PLACE,
+                                 BIRTH_COUNTRY_CODE, CREATE_DATE, LAST_NAME_KEY, ALIAS_OFFENDER_ID, FIRST_NAME_KEY,
+                                 MIDDLE_NAME_KEY, OFFENDER_ID_DISPLAY, ROOT_OFFENDER_ID, CASELOAD_TYPE, MODIFY_USER_ID,
+                                 MODIFY_DATETIME, ALIAS_NAME_TYPE, PARENT_OFFENDER_ID, UNIQUE_OBLIGATION_FLAG,
+                                 SUSPENDED_FLAG, SUSPENDED_DATE, RACE_CODE, REMARK_CODE, ADD_INFO_CODE, BIRTH_COUNTY,
+                                 BIRTH_STATE, MIDDLE_NAME_2, TITLE, AGE, CREATE_USER_ID, LAST_NAME_ALPHA_KEY,
+                                 CREATE_DATETIME, NAME_SEQUENCE, AUDIT_TIMESTAMP, AUDIT_USER_ID, AUDIT_MODULE_NAME,
+                                 AUDIT_CLIENT_USER_ID, AUDIT_CLIENT_IP_ADDRESS, AUDIT_CLIENT_WORKSTATION_NAME,
+                                 AUDIT_ADDITIONAL_INFO)
+VALUES (2554472, null, 'SEQ', 'AIMONIS', null, 'EF''LIAICO', 'URUA', TO_DATE('1987-08-07', 'YYYY-MM-DD'), 'M',
+        null, 'A552', 'SHEFFIELD', 'ENG', TO_DATE('2017-02-17', 'YYYY-MM-DD'), 'AIMONIS', null, 'EFLIAICO',
+        'URUA', 'G6942UN', 2554472, 'INST', 'OMS_OWNER',
+        TO_DATE('2017-06-26', 'YYYY-MM-DD'), 'CN', null, null, null, null, 'W1',
+        null, null, null, null, null, 'MR', null, 'QQM25S', 'A',
+        TO_DATE('2017-02-17', 'YYYY-MM-DD'), '1234',
+        TO_DATE('2017-06-26', 'YYYY-MM-DD'), 'OMS_OWNER', 'MERGE', 'pgrant',
+        '10.102.1.69', 'unknown', null);
+
+INSERT INTO AGENCY_LOCATIONS (AGY_LOC_ID, DESCRIPTION, AGENCY_LOCATION_TYPE, ACTIVE_FLAG)
```

Comment:
I think you should use one of the AGENCY_LOCATIONS that have been created by the flyway scripts.

Reply:
done